src/components/Card/Card.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from "react";
+
+import cn from "classnames";
+import CardHeader from "./CardHeader";
+import CardTitle from "./CardTitle";
+import CardBody from "./CardBody";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +title?: string,
+  +body?: React.Node,
+  +as: React.ElementType
+|};
+
+const Card = ({
+  className,
+  as: Component,
+  title,
+  body,
+  ...props
+}: Props): React.Node => {
+  const classes = cn("card", className);
+  const card_header =
+    title === "" ? null : (
+      <CardHeader>
+        <CardTitle>{title}</CardTitle>
+      </CardHeader>
+    );
+  const card_body = body === null ? null : <CardBody>{body}</CardBody>;
+
+  if (card_header !== null || card_body !== null) {
+    return (
+      <Component className={classes} {...props}>
```

Comment:
Can we find an alternative way we can implement it rather than remove it? To give some context on the 'as' prop, it's only found on components that may need to switch their child elements type.

For example, on the Tabler form elements demo page, the page content is a card but rather than a <div> a <form> is the output, but it should still retain all the same properties, features and classes as a card https://jonthomp.github.io/react-tabler-components/form-elements

Reply:
Ahh I see, thanks for the explanation!  In reality, we should have the styles written so that it doesn't require us to do this, but I'd prefer not to rewrite those, at least not for now :)  Instead, lets agree to try to only use this wherever necessary?  And maybe we could rename it to like "rootComponentType" or something a little more descriptive?  Just trying to make it as easy to read/understand as possible

Reply:
Yeah totally agree, although what do you think to it being named RootComponent, capitalized to emphasise it should be a react element and removing the 'Type' because you are passing through an element rather than just its type?

Also with the removing of defaultProps how about implementing this prop slightly differently? The type would be `+RootComponent?: React.ElementType` then the component:
```
function Card({ RootComponent, ...props }: Props): React.Node {
  const Component = RootComponent || "div";
  return <Component {...props} />;
}
```

Reply:
That sounds good to me!  I like the `RootComponent || "div"` - very easy to read and understand even for beginners.